scripts/validate_docstrings.py

```diff
@@ -446,134 +450,157 @@ def validate_one(func_name):
     dict
         A dictionary containing all the information obtained from validating
         the docstring.
+
+    Notes
+    -----
+    The errors are codified in the next way:
+    - First two characters: Section where the error happens:
+       * GL: Global (no section, like section ordering errors)
+       * SS: Short summary
+       * ES: Extended summary
+       * PR: Parameters
+       * RT: Returns
+       * YD: Yields
+       * RS: Raises
+       * WN: Warns
+       * SA: See Also
+       * NT: Notes
+       * RF: References
+       * EX: Examples
+    - Last two characters: Numeric error code inside the section
+
+    For example, RT02 is the second codified error in the Returns section.
```

Comment:
Is `RT02` an actual error code?  Might instead want to use a real one as an example (and provide the semantics behind the given error code) as a more substantive example?